geoapps/utils/models.py

```diff
@@ -20,6 +25,26 @@
 from geoapps.utils.surveys import compute_alongline_distance
 
 
+def face_average(mesh: TreeMesh, model: np.ndarray) -> np.ndarray:
+    """
+    Compute the average face values of a model
+
+    :param mesh: Tree mesh object
+    :param model: A vector of cell centered property values
+    """
+    return mesh.stencil_cell_gradient.T * (mesh.stencil_cell_gradient * model)
+
+
+def floating_active(mesh: TreeMesh, active: np.ndarray):
+    """
+    True if there are any active cells in the air
+
+    :param mesh: Tree mesh object
+    :param active: active cells array
+    """
+    return True if 6 in face_average(mesh, active) else False
```

Comment:
That's cool, didn't know we could do 6 in. No issues with integer and float array?